Gravity/Gravity/Base/BaseDto.cs

```diff
@@ -187,7 +137,15 @@ private static object ConvertPropertyValue(PropertyInfo property, RdoFieldType f
 				case RdoFieldType.MultipleObject:
 					{
 						return new FieldValueList<Artifact>(
-							((IList<int>)propertyValue).Select(x => new Artifact(x)));
+							(
```

Comment:
Cast objects to BaseDto instead of using reflection (yes, this issue preceded you). Also align parentheses better

Reply:
very good call.  I had the same thought about reflection being expensive and taking a pass through the code looking for ways to optimize or potentially using caching or something.  No need to continue to contribute to the problem.  Done